src/core-components/Modal/Modal.less

```diff
@@ -0,0 +1,51 @@
+@import './src/assets/styles/variables/colors.less';
+
+.ant-modal-content {
+  background-color: transparent;
+}
+
+.ant-modal-header {
+  padding: 24px 16px 0;
+  background-color: @color-gray-1;
+  border-radius: 16px 16px 0 0;
+}
+
+.ant-modal-body {
+  padding: 24px 16px;
+  background-color: @color-gray-1;
+  color: @color-gray-9;
+}
+
+.ant-modal-footer {
+  padding: 0 16px 24px;
```

Comment:
```padding: 0 16px 24px 0;```
I think it is more correct.

Reply:
Stylelint will through an Error on it, I tried